mod/groups/languages/en.php

```diff
@@ -134,6 +134,7 @@
 	'groups:joinrequestmade' => 'Requested to join group',
 	'groups:joined' => 'Successfully joined group!',
 	'groups:left' => 'Successfully left group',
+	'groups:requestexists' => 'You already requested join this group.',
```

Comment:
I think proper English is `You have already requested` if no time is specified. Past tense is used if you specify time `You requested to joint his group 2 days ago`.